posts/2021-08-31-liberty-bikes.adoc

```diff
@@ -0,0 +1,76 @@
+blog_description: See how a multiplayer browser game can be implemented as microservices with a Open Liberty and MicroProfile.
+open-graph-image: https://openliberty.io/img/blog/lb-title.png
+---
+= Liberty Bikes: A Microservices-Based Game Built On Open Liberty
+Liam Westby <https://github.com/realModusOperandi>
+:imagesdir: /
+:url-prefix: https://openliberty.io
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+[#Intro]
+
+image::img/blog/lb-title.png[Liberty Bikes, built on Open Liberty, align="center"]
+
+A while ago, the Open Liberty team was looking for some cool and interesting demos: projects built to take advantage of and show off some of the new features in Open Liberty, including MicroProfile. Demos that could help illustrate key concepts provided by this great technology and could be used as tutorials to show developers how to do cool things in their own code.  That sounded like a fun project! A team of us (Olen Cook, Ryan Esch, Andy Guibert, and I, with art by Sarah Ishida) got together to come up with a submission. 
+
+But what to create? It should be more complex and sophisticated than Hello World, as well as being polished and interesting enough to draw people in who might be passing by a conference booth. Something that keeps bringing people back, not just to have a look at the code, but to keep using it again and again. 
+
+Something like... a game! We call it Liberty Bikes, and you can play it right here: link:http://libertybikes.mybluemix.net[Liberty Bikes]. It's all open source, so you can see how it works here: link:https://github.com/OpenLiberty/liberty-bikes[Liberty Bikes on GitHub]
+
+[#About]
+== What Is Liberty Bikes?
+
+image::img/blog/lb-game-screen.png[screen capture of the Liberty Bikes game board, align="center"]
+
+link:http://libertybikes.mybluemix.net[Liberty Bikes] is a multiplayer game for up to four players (either on four separate computers or with four mobile device controllers and one hosting computer) in which you pilot a bike that leaves a trail as it moves around the arena. Your objective is to be the last player standing, by avoiding other players' trails, the walls, and the obstacles scattered throughout the arena. You can steer your bike left or right, but you can't slow down or stop. 
+
+The game can be played in two ways: if you are on a desktop or laptop, you'll be able to see the game board and steer using the arrow keys. If you are on a mobile device, you'll get a touch-based gamepad, but you'll need to use a computer hosting the game as a spectator in order to see the board. 
+
+image::img/blog/lb-mobile-gamepad.png[screen capture of the Liberty Bikes mobile game pad, align="center"]
+
+Some player statistics, such as wins and rounds played, are stored for each player. You can see how you're doing as you play, and who among you is the best at staying in the game. You can play either as a guest or by signing in through one of several OAuth providers.
```

Comment:
recommend not splitting the clauses, e.g:

Some player statistics are stored for each player, such as wins and rounds played. As you play, you can see how you're doing  and who among you is the best at staying in the game.